x-pack/plugins/integration_assistant/server/graphs/log_type_detection/graph.ts

```diff
@@ -58,15 +79,16 @@ function modelOutput({ state }: LogDetectionBaseNodeParams): Partial<LogFormatDe
     lastExecutedChain: 'modelOutput',
     results: {
       samplesFormat: state.samplesFormat,
-      parsedSamples: state.logSamples, // TODO: Add parsed samples
+      parsedSamples: state.jsonSamples,
+      additionalProcessors: state.additionalProcessors,
     },
   };
 }
 
 function logFormatRouter({ state }: LogDetectionBaseNodeParams): string {
-  // if (state.samplesFormat === LogFormat.STRUCTURED) {
-  //   return 'structured';
-  // }
+  if (state.samplesFormat.name === 'structured') {
```

Comment:
should we continue to use the constants? 

```suggestion
  if (state.samplesFormat.name === LogFormat.STRUCTURED) {
```

Reply:
Probably yeah... That looks clean.